src/Main.elm

```diff
@@ -1,10 +1,362 @@
 module Main exposing (main)
 
-import Data.Audience
-import Data.AudienceFolder
-import Html exposing (Html)
+import Browser
+import Data.Audience as Audience
+import Data.AudienceFolder as AudienceFolder
+import Decoder
+import Html exposing (Html, button, div, text, ul)
+import Html.Attributes exposing (class)
+import Html.Events exposing (onClick)
+import Http exposing (Error(..))
 
 
-main : Html msg
+type Msg
+    = MsgGotResultsAudienceFolder (Result Http.Error (List AudienceFolder.AudienceFolder))
+    | MsgGotResultsAudience (Result Http.Error (List Audience.Audience))
+    | OpenFolder Int
+    | GoBack
+    | SelectCategory Audience.AudienceType
+
+
+type alias Model =
+    { audienceFolder : List AudienceFolder.AudienceFolder
+    , audience : List Audience.Audience
+    , urlAudienceFolder : String
+    , urlAudience : String
+    , errorMessage : Maybe String
+    , currentFolderId : Maybe Int
+    , selectedCategory : Audience.AudienceType
+    }
+
+
+main : Program () Model Msg
 main =
-    Html.text "There will be app soon!"
+    Browser.element
```

Comment:
Very nitpicking but [`Browser.element`](https://package.elm-lang.org/packages/elm/browser/latest/Browser#element) is actually for HTML elements managed by Elm that you can embed into any JavaScript project. [`Browser.document`](https://package.elm-lang.org/packages/elm/browser/latest/Browser#document) would be more correct, since yours is a whole application done in Elm. But **[`Browser.element`](https://package.elm-lang.org/packages/elm/browser/latest/Browser#element) gets it done so it's okay.**